pkg/crc/config/viper_config.go

```diff
@@ -39,34 +39,59 @@ func NewViperStorage(configFile, envPrefix string) (*ViperStorage, error) {
 	}, nil
 }
 
-func (c *ViperStorage) Get(key string) interface{} {
-	return c.viper.Get(key)
+func (c *ViperStorage) Get(key string) (interface{}, error) {
+	if err := c.refresh(); err != nil {
+		return nil, err
+	}
```

Comment:
disk IO + parsinng everytime we need a config value seems bad from an efficiency point of view?

Reply:
See https://github.com/code-ready/crc/pull/1837#discussion_r557477597 , only a temporary step towards a better fix.

Reply:
It's now removed